### ISSUE TITLE
do word counts and other metrics once the full corpus is available

### ISSUE THREAD

**Issue by JonathanReeve** [Git-Lit/git-lit#10]
Suggested by Heather Froehlich here: https://twitter.com/heatherfro/status/643519126917541888

**Comment by tfmorris** [comment]
I agree metrics are critically important and have already started on a script to extract them, but word count seems like an unusual first choice (unless you're writing a press release).  It won't matter how many words there are if all of them are garbage, so my initial focus is on quality metrics like OCR word confidence distribution and meta metrics like diversity of preprocessing & OCR pipelines, diversity of fonts and languages, etc.

**Comment by JonathanReeve** [comment]
I can see how word count might be interesting to corpus linguists, but I'm super interested in things like language distribution, too. I can't wait to see what your script reveals.

**Comment by tfmorris** [comment]
A first cut of this is available in ea401dd, although the stats aren't recorded anywhere.  It might make sense to put a summary in the README in addition to keeping a master CSV/TSV.

Current stats include:
- average word confidence score for volume (with sanity check to make sure page confidence recorded in ALTO matches average word confidence for page)
- character confidence (0-10) frequency histogram
- page count
- word count
- hyphenation point count (beginning & ending, with warnings for mismatched count)
- cursory paragraph & text style counts

The metadata files don't contain language, so it will need to be derived separately.
